docs/troubleshooting/node-debug.md

```diff
@@ -0,0 +1,89 @@
+
+`Current Sync Status: NoSync (#64817610 DakWGcEUsNpRSXEgBm4FUGEqnJy9qVKouB8V9ek8BqYr)`
+- Current Sync Status: NoSync means the node is synced and not syncing. What follows is the block number and block hash. Other status that can be shown are header sync and block sync.
+
+`Number of peers: 30/40`
+This means there are 30 active peers. 40 is the max number of peers.
+
+`Validators: 100 Known: 100 Reachable: 100`
+- `Validators: 100`: is determined in the last epoch by the network.
+- `Known: 100`: is that the current node has the validator accountID present in this node's database.
+- `Reachable: 100`: If a validator is not reachable from the current node, it will not be included in the count.
+
+What follows is a table of the node peers.
+- `Address`: node IP and port.
+- `Validator?`: Shows if the node is a validator. If it is, it shows the pool name.
+- `AccountID`: The first line shows the account id for the peer. The second line shows the peer id.
+- `Height`: shows the latest block height.
+- `Tracked Shards`: shows which shards the node is tracking.
+- `Archival`: shows whether the node is archival or not.
+- `Route to validators`: List of validators, the paths to which contain this current node as a peer.
+
+### Chain Info
+This page displays the current point-in-time status of the node in terms of orphan blocks and missing chunks.
+
+`Current head: 3mivFUczgGWM6RvwvaT2oQ93zRQQTn3riXNtWxByntD1 @ 64818564`: The current head is the latest block of this node.
+
+`Current header head: 3mivFUczgGWM6RvwvaT2oQ93zRQQTn3riXNtWxByntD1 @ 64818564`:
+The header_head is the latest 'header' that we know of. If node is up-to-date, then head is equal to header_head. If the node is syncing, the header_head may be larger than the head.
+
+`Orphan Pool`: This table shows a list of orphan blocks for this current node. The block hash and block height are listed. (e.g. It is possible that this node received a block for which its predecessor is unknown. Once the predecessor is known, the orphan block would no longer be in the pool.)
+
+`Missing Chunks Pool`
+This table shows a list of blocks that are missing some of its chunks in this node. The block hash and block height are listed. As your node request for missed chunks, you can refresh the page to see the latest status and to see if the chunk has been received and stored. If a chunk has been no longer missing, it will be removed from this table.
+
+
+### Epoch Info
+
+The epoch displays the current epoch and the most recent five epoches.
+
+- `Epoch id`: block hash of the final block in epoch T-2.
+- `Start height`: the first block of the epoch.
+- `Protocol version`: The protocol version of the epoch.
+- `First block`: Block hash of the first block in the epoch.
+- `Epoch start`: How long ago the epoch has started.
+
+The table below displays the validator history in the current epoch and the last five epochs (represented by the epoch hash) with either TRUE or FALSE.  
+
+
```

Comment:
we could also mention, that they can see the example here:
http://34.147.91.25:3030/debug

(but we might want to get a 'better URL' in the future, as this IP can change)